tests/sentry/mail/activity/test_release_summary.py

```diff
@@ -52,7 +56,14 @@ def test_simple(self):
                     data={"version": self.release.version, "deploy_id": self.deploy.id},
                 )
             )
-
+        # opt-in to getting active_release notifications
+        NotificationSetting.objects.update_settings(
+            ExternalProviders.EMAIL,
+            NotificationSettingTypes.ACTIVE_RELEASE,
+            NotificationSettingOptionValues.ALWAYS,
+            user=self.user1,
+            project=self.project,
+        )
```

Comment:
This is now needed since the default value for `NotificationSettingTypes.ACTIVE_RELEASE` is `NotificationSettingOptionValues.NEVER` which means don't send notification to user unless they explicitly opt-in (db row exists in `NotificationSetting` for this user and provider).